app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -0,0 +1,325 @@
+import { useContext, useEffect } from 'react';
+import type { ReactNode } from 'react';
+import { useI18n } from '@18f/identity-react-i18n';
+import { useImmutableCallback } from '@18f/identity-react-hooks';
+import AcuantContext from '../context/acuant';
+
+declare let AcuantCameraUI: AcuantCameraUIInterface;
+declare global {
+  interface Window {
+    AcuantCameraUI: AcuantCameraUIInterface;
+  }
+}
+
+/**
+ * Type definition for export only
+ */
+type AcuantGlobals = {
+  AcuantCameraUI: AcuantCameraUIInterface;
+  AcuantCamera: AcuantCameraInterface;
+};
+type AcuantGlobal = Window & AcuantGlobals;
+
+enum AcuantDocumentStateEnum {
+  NO_DOCUMENT = 0,
+  SMALL_DOCUMENT = 1,
+  BIG_DOCUMENT = 2,
+  GOOD_DOCUMENT = 3,
+}
+
+/**
+ * @enum {number}
+ */
+export const AcuantDocumentState = {
+  NO_DOCUMENT: 0,
+  SMALL_DOCUMENT: 1,
+  BIG_DOCUMENT: 2,
+  GOOD_DOCUMENT: 3,
+};
+
+enum AcuantUIStateEnum {
+  CAPTURING = -1,
+  TAP_TO_CAPTURE = -2,
+}
+
+/**
+ * @enum {number}
+ */
+export const AcuantUIState = {
+  CAPTURING: -1,
+  TAP_TO_CAPTURE: -2,
+};
+
+type AcuantFrameState = AcuantDocumentStateEnum & AcuantUIStateEnum;
+
+type AcuantCaptureType = 'AUTO' | 'TAP';
+
+/**
+ * @prop NONE No document detected.
+ * @prop SMALL_DOCUMENT Document does not fill frame.
+ * @prop BIG_DOCUMENT Document is too close to the frame.
+ * @prop GOOD_DOCUMENT Document is good and capture is pending.
+ * @prop CAPTURING Document is being captured.
+ * @prop TAP_TO_CAPTURE Explicit user action to capture after delay.
+ */
+interface AcuantCameraUIText {
+  NONE: string;
+  SMALL_DOCUMENT: string;
+  BIG_DOCUMENT: string;
+  GOOD_DOCUMENT: string | null;
+  CAPTURING: string;
+  TAP_TO_CAPTURE: string;
+}
```

Comment:
I think we had chatted about this briefly in our pairing last week, but while JSDoc tags carry over well enough for function `@param` documentation, I don't know that `@prop` would work the same way in documenting an interface like this. Instead, I know that this form _does_ work (and VSCode will pick up the associated description in its tooltips).

```suggestion
interface AcuantCameraUIText {
  /**
   * No document detected.
   */
  NONE: string;

  /**
   * Document does not fill frame.
   */
  SMALL_DOCUMENT: string;

  /**
   * Document is too close to the frame.
   */
  BIG_DOCUMENT: string;

  /**
   * Document is good and capture is pending.
   */
  GOOD_DOCUMENT: string | null;

  /**
   * Document is being captured.
   */
  CAPTURING: string;

  /**
   * Explicit user action to capture after delay.
   */
  TAP_TO_CAPTURE: string;
}
```